src/CloudNimble.DotNetDocs.Sdk.Tasks/CloudNimble.DotNetDocs.Sdk.Tasks.csproj

```diff
@@ -1,54 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This whole project went away because you can use a few features of the .NET SDK to both develop the tasks and bundle them in the SDK package altogether.